src/app_engine/apprtc.py

```diff
@@ -119,7 +119,22 @@ def get_wss_parameters(request):
   ws_tls = request.get('wstls')
 
   if not ws_host_port_pair:
-    ws_host_port_pair = constants.WSS_HOST_PORT_PAIR
+    memcache_client = memcache.Client()
```

Comment:
Ok. Reason I asked was more because with ndb could've added filter / order in query so we don't have to do sorting stuff below.